src/scripts/core/models/column.ts

```diff
@@ -14,85 +15,90 @@ export class Column extends Container{
         this.columnID = columnID;
         this.columnSize = columnSize;
         this.availHeight = availHeight;
-        for(var i = 0; i < this.columnSize; i++) {
-            const newToken = new Token(this.columnID, i, this.columnSize, availWidth, availHeight);
-            newToken.y = (availHeight / (this.columnSize/i));
+        for (var tokenIndex: number = 0; tokenIndex < this.columnSize; tokenIndex++) {
+            const newToken = new Token(this.availHeight, this.availHeight, this.columnID, tokenIndex, this.columnSize);
```

Comment:
The constructor takes a width and a height and you have passed through height twice, is this intentional?